backend/src/packages/chaiNNer_standard/utility/color/split_color.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+
+from enum import Enum
+
+from nodes.impl.color.color import Color
+from nodes.properties.inputs import ColorInput, EnumInput
+from nodes.properties.outputs import NumberOutput
+
+from .. import color_group
+
+
+def _norm(n: float) -> int:
+    return int(max(min(n * 255, 255), 0))
+
+
+class SplitColorMode(Enum):
+    NUMBER = 0
+    COLOR = 1
+
+
+output_navi_type = """match Input1 {
+    SplitColorMode::Number => int(0..255),
+    SplitColorMode::Color => Color { channels: 1 },
+    _ => never }
+"""
+
+
+@color_group.register(
+    schema_id="chainner:utility:split_color",
+    name="Split Color",
```

Comment:
IMO this node should be called "Separate Color" to be consistent with the Separate RGBA node.